guides/common/modules/ref_storage-guidelines.adoc

```diff
@@ -9,9 +9,8 @@ This is a requirement for the `puppetserver` service to work.
 
 * Because most {ProductName} data is stored in the `/var` directory, mounting `/var` on LVM storage can help the system to scale.
 
-* Use high-bandwidth, low-latency storage for the `/var/lib/pulp/` directories.
+* Use high-bandwidth, low-latency storage for the `/var/lib/pulp/` and PostgreSQL `/var/lib/pgsql/` directories.
```

Comment:
Thanks for @maximiliankolb for reminding me we need to check the path. There's actually an attribute for it that has a different value on Debian.
```suggestion
* Use high-bandwidth, low-latency storage for the `/var/lib/pulp/` and PostgreSQL `{postgresql-lib-dir}` directories.
```